opennms-base-assembly/src/main/filtered/bin/fix-permissions

```diff
@@ -66,6 +66,15 @@ fi
 
 SUDO="$(command -v sudo 2>/dev/null || which sudo 2>/dev/null || :)"
 
+do_sudo() {
+  if [ -x "$SUDO" ]; then
+    "$SUDO" "$@"
+  else
+    # if no sudo, fall back to running and hope for the best
+    "$@"
```

Comment:
Maybe we can give the user a warning that we run as $user here and we are in a fallback mode and expect having sudo available?